firedantic/_async/indexes.py

```diff
@@ -0,0 +1,140 @@
+from logging import getLogger
+from typing import Iterable, List, Optional, Set, Type
+
+from google.api_core.operation_async import AsyncOperation
+from google.cloud.firestore_admin_v1 import (
+    CreateIndexRequest,
+    Index,
+    ListIndexesRequest,
+)
+from google.cloud.firestore_admin_v1.services.firestore_admin import (
+    FirestoreAdminAsyncClient,
+)
+
+from firedantic._async.model import AsyncBareModel, IndexDef, IndexField
+from firedantic._async.ttl_policy import set_up_ttl_policies
+
+logger = getLogger("firedantic")
+
+
+async def list_existing_indexes(
```

Comment:
Since this returns a set, I'd rather see this called `get_existing_indexes`, as the `list_...` in the name makes me thing it's returning a list.